### ISSUE TITLE
Add support for new released Camera

### ISSUE THREAD

**Issue by smartmatic** [OpenWonderLabs/homebridge-switchbot-openapi#268]
**Describe the solution you'd like**

Hello,

i have a the new released SwitchBot camera at home and it would be really nice if the camera can be added in order to work with the plugin. The device discovery text is added in the related section.

Is that possible?

**Additional context**

[23/08/2021, 12:13:47] [SwitchBot] {"deviceId":"xxxxxxx","deviceName":"GarageCam","enableCloudService":true,"hubDeviceId":"xxxxxx"}
[23/08/2021, 12:13:47] [SwitchBot] Device: GarageCam with Device Type: undefined, is currently not supported. Submit Feature Requests Here: https://git.io/JL14Z
[23/08/2021, 12:13:47] [SwitchBot] deviceStatus - Hub Mini 3C {"statusCode":100,"body":{},"message":"success"}
[23/08/2021, 12:13:47] [SwitchBot] deviceStatus - GarageCam {"statusCode":100,"body":{},"message":"success"}

**Comment by donavanbecker** [comment]
@smartmatic This isn't not integrated into the api yet, but it does look like it is a Tuya camera. But none of the Tuya plugins work with cameras yet.

**Comment by smartmatic** [comment]
yes, it is a tuya based camera. I was able to add and connect the camera with the Smart Live App.
